rooch-da/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+#[cfg(test)]
+mod test;
+
+// Request to store data to the DA server
+pub struct DAStoreRequest {
+    pub message: Vec<u8>,
+    pub timeout: u64,
+    pub signature: Vec<u8>,
+}
+
+// Data storage response returned by the DA server
+pub struct DAStoreResponse {
+    // The hash of multi-signature collection returned by multiple nodes comprising the DA committee.
+    // The actual public key collection is stored in DA and retrieved from it as needed for verification.
+    pub keyset_hash: Vec<u8>,
+    // Hash of the data returned by the DA Server.
+    pub data_hash: Vec<u8>,
+    // The timeout period for the storage backend.
+    pub timeout: u64,
+    // Use signersMask to extract the corresponding public key set from the keyset data.
+    pub signers_mask: u64,
+    // Multi-signature of the DA committee.
+    pub sig: Vec<u8>,
+    // The version of this struct.
+    pub version: u8,
+}
+
+pub struct DAGetRequest {
+    pub hash: Vec<u8>,
+}
+
+pub struct DAGetResponse {
+    pub data: Vec<u8>,
+}
+
+pub struct DAStatusResponse {
+    pub status_code: u64,
+    pub status_data: Vec<u8>,
+}
+
+pub trait DAClientMethods {
```

Comment:
We've discussed about APIs here and in meetings:

https://github.com/rooch-network/rooch/issues/85

Pls trying to update codes according to results we've achieved

Reply:
The problem has been fixed based on the previous records.